server/index.js

```diff
@@ -0,0 +1,16 @@
+const Koa = require('koa');
+const bodyParser = require('koa-bodyparser');
+const cors = require('koa-cors');
+const router = require('./router');
+
+
+const app = new Koa();
+const port = 4100;
+
+app
+  .use(bodyParser())
+  .use(cors())
+  .use(router.routes());
+
+app.listen(port, () => `Server running at port ${port}`);
```

Comment:
your are not logging `Server running at port ${port}`
cost mit 20 minutes because I thought the server is not running.

Reply:
cool! i missed that. thx a lot